### ISSUE TITLE
Create miRNA-Seq expression matrix

### ISSUE THREAD

**Issue by jharenza** [PediatricOpenTargets/OpenPBTA-analysis#422]
#### What data file(s) does this issue pertain to?
```
mirna-expression-htgseq-counts.tsv.gz
mirna-expression-htgseq-cpm.tsv.gz
mirna-expression-htgseq-normalized-counts.tsv.gz
```

#### What release are you using?

For v13

#### Put your question or report your issue here.
We will release miRNA expression data from the [HTGSeq platform](https://www.htgmolecular.com/assays/mirna-wta) performed on 262 PBTA samples.

The matrices need to be regenerated by @mkoptyra using the latest version of the HTGSeq parser.

We also have an outstanding question as to why we see negative CPM in these files - @mkoptyra emailed a scientist at the company.

cc @zzgeng

**Comment by jharenza** [comment]
This is blocked pending a software rerun

**Comment by jharenza** [comment]
closed with #545
